.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+---
+blank_issues_enabled: true
+contact_links:
+  -   name: 👮 Report a security issue
+      url: 'mailto:pt-frontend-infrastructure@liferay.com'
+      about: Report security problems privately here
```

Comment:
I don't know what this looks like in the UI and I won't until we merge it, but apparently, [this is a thing](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository).

Reply:
I also don't know whether `mailto:` URLs will work, because the examples in the docs don't mention it.